src/validation-schemas/user-schema.js

```diff
@@ -0,0 +1,58 @@
+'use strict';
+
+const Joi = require(`joi`);
+const {REGISTRATION_MESSAGE} = require(`../constants`);
+
+module.exports = Joi.object({
+  firstName: Joi.string()
+    .required()
+    .max(50)
+    .pattern(new RegExp(`^[a-zA-Zа-яА-Я ]{0,50}$`))
```

Comment:
`\s` вместо пробела лучше использовать
`a-zA-Z` эквивалентно `a-z` + флаг `i`
```suggestion
    .pattern(new RegExp(`^[a-zа-я\s]{0,50}$`, 'i'))
```

Reply:
Поправил. Взял предложенный вариант.